src/Avalonia.NameGenerator/Generator/InitializeComponentCodeGenerator.cs

```diff
@@ -6,19 +6,35 @@ namespace Avalonia.NameGenerator.Generator
 {
     internal class InitializeComponentCodeGenerator: ICodeGenerator
     {
-        private readonly bool _attachDevTools;
+        private readonly bool _diagnosticsAreConnected;
         private const string AttachDevToolsCodeBlock = @"
 #if DEBUG
-            this.AttachDevTools();
+            if(attachDevTools)
```

Comment:
nit: white space in order to improve generated sources readability 
```suggestion
            if (attachDevTools)
```